master/styling.html

```diff
@@ -464,8 +464,6 @@ <h2 id="PresentationAttributes">Presentation attributes</h2>
       <a>'shape-rendering'</a>,
       <a>'solid-color'</a>,
       <a>'solid-opacity'</a>,
-      <a>'stop-color'</a>,
-      <a>'stop-opacity'</a>,
```

Comment:
I think the wrong properties were removed here. Did you mean to remove `solid-color` and `solid-opacity` instead of `stop-color` and `stop-opacity`?